libraries/apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/internal/DefaultApolloExtension.kt

```diff
@@ -600,7 +601,7 @@ abstract class DefaultApolloExtension(
         codegenMetadataConsumerConfiguration.dependencies.add(it)
       }
 
-      val pending = pendingDownstreamDependencies.get(name)
+      val pending = pendingDownstreamDependencies.get(service.name)
```

Comment:
This is an unrelated fix that I discovered by serendipity for `isADependencyOf(bidirectional: true)`